src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -86,11 +112,133 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
+    );
+    expect((await screen.findAllByRole('radio')).length).to.equal(12);
+
+    userEvent.click(await screen.findByText(/\+ 5 more providers/i));
+    await waitFor(() =>
+      screen.getByText(/displaying 1 to 15 of 16 providers/i),
+    );
+    expect((await screen.findAllByRole('radio')).length).to.equal(17);
+
+    userEvent.click(await screen.findByText(/\+ 1 more providers/i));
+    await waitFor(() =>
+      screen.getByText(/displaying 1 to 16 of 16 providers/i),
+    );
+    expect((await screen.findAllByRole('radio')).length).to.equal(18);
+
+    // Choose Provider
+    userEvent.click((await screen.findAllByRole('radio'))[3]);
+    userEvent.click(
+      await waitFor(() =>
+        screen.getByRole('button', { name: /choose provider/i }),
+      ),
+    );
+    expect(screen.baseElement).to.contain.text(
+      'AJADI, ADEDIWURA700 CONSTITUTION AVE NEWASHINGTON, DC 20002-65999349.3 miles',
+    );
+
+    // Change Provider
+    userEvent.click(
+      await screen.getByRole('button', { name: /change provider/i }),
+    );
+    userEvent.click((await screen.findAllByRole('radio'))[4]);
+    userEvent.click(
+      await waitFor(() =>
+        screen.getByRole('button', { name: /choose provider/i }),
+      ),
+    );
+    expect(screen.baseElement).to.contain.text(
+      'LYONS, KRISTYN1785 S HAYES STARLINGTON, VA 22202-27149347.1 miles',
+    );
+
+    // Cancel Selection (not clearing of a selected provider)
+    userEvent.click(
+      await screen.getByRole('button', { name: /change provider/i }),
+    );
+    await waitFor(() => screen.getByText(/displaying 1 to 5 of 16 providers/i));
+    await waitFor(() => screen.getByRole('button', { name: /cancel/i }));
+    expect(screen.baseElement).to.contain.text(
+      'LYONS, KRISTYN1785 S HAYES STARLINGTON, VA 22202-27149347.1 miles',
+    );
+  });
+  // TODO: show more
+  // TODO: provider fetch failure
+  it('should display an error when choose a provider clicked and provider fetch error', async () => {
+    const store = createTestStore(initialState);
+    await setTypeOfCare(store, /primary care/i);
+    await setTypeOfFacility(store, /Community Care/i);
+    mockCCProviderFetch(
+      initialState.user.profile.vapContactInfo.residentialAddress,
+      ['208D00000X', '207R00000X', '261QP2300X'],
+      calculateBoundingBox(
+        initialState.user.profile.vapContactInfo.residentialAddress.latitude,
+        initialState.user.profile.vapContactInfo.residentialAddress.longitude,
+        60,
+      ),
+      CC_PROVIDERS_DATA,
+      true,
+    );
+    const screen = renderWithStoreAndRouter(
+      <CommunityCareProviderSelectionPage />,
+      {
+        store,
+      },
+    );
+
+    // Trigger provider list loading
+    userEvent.click(await screen.findByText(/Choose a provider/i));
+    expect(
+      await screen.getByRole('heading', {
+        name: /we’re sorry\. we’ve run into a problem/i,
+      }),
+    );
+    expect(
+      await screen.getByText(
```

Comment:
These should use `findByText` instead of `getByText`, since you're using `await`.

Reply:
Fixed